apps/cookbook/.eslintrc.json

```diff
@@ -1,26 +0,0 @@
-{
```

Comment:
all `.eslintrc.json` files have been automatically migrated to `eslint.config.mjs` as per https://eslint.org/docs/latest/use/configure/migration-guide